modules/Planner/planner_view_full_myHomeworkProcess.php

```diff
@@ -63,8 +63,7 @@
         } else {
             //Get variables
             $homework = $_POST['homework'];
-            if ($_POST['homework'] == 'Yes') {
-                $homework = 'Y';
+            if ($homework == 'Y') {
```

Comment:
The conditional that has been removed uses "Yes" but the new one uses "Y". I guess there is a corresponding change in the front end?